i18n/zh-cn/docusaurus-plugin-content-docs/current/quick-start/opensergo-control-plane.md

```diff
@@ -12,12 +12,34 @@ sidebar_position: 1
 
 ## 在集群中安装 OpenSergo 控制平面
 
-TODO: helm chart
+```shell
+# 创建 opensergo-system 命名空间
+kubectl apply -f opensergo-control-plane/k8s/namespace.yaml
+
+# opensergo 控制面相关权限配置
+kubectl apply -f opensergo-control-plane/k8s/rbac/role.yaml
+kubectl apply -f opensergo-control-plane/k8s/rbac/rolebinding.yaml
+kubectl apply -f opensergo-control-plane/k8s/rbac/serviceaccounts.yaml
+
+# 安装 opensergo spec crd
+kubectl apply -f opensergo-control-plane/k8s/crd/bases/fault-tolerance.opensergo.io_circuitbreakerstrategies.yaml
+kubectl apply -f opensergo-control-plane/k8s/crd/bases/fault-tolerance.opensergo.io_concurrencylimitstrategies.yaml
+kubectl apply -f opensergo-control-plane/k8s/crd/bases/fault-tolerance.opensergo.io_faulttolerancerules.yaml
+kubectl apply -f opensergo-control-plane/k8s/crd/bases/fault-tolerance.opensergo.io_ratelimitstrategies.yaml
+kubectl apply -f opensergo-control-plane/k8s/crd/bases/fault-tolerance.opensergo.io_throttlingstrategies.yaml
+kubectl apply -f opensergo-control-plane/k8s/crd/bases/traffic.opensergo.io_trafficerouters.yaml
+
+# 安装 opensergo 控制平面 deployment
+kubectl apply -f opensergo-control-plane/k8s/deployment.yaml
+
+# 创建 opensergo 控制平面 service
+kubectl apply -f opensergo-control-plane/k8s/service.yaml
```

Comment:
need update to match new yaml file 
<img width="457" alt="image" src="https://user-images.githubusercontent.com/43714056/211732820-f1e99a90-b19a-4f5a-9314-76088840b2fd.png">